slashCommands/narratorSlash/sroleSlash.js

```diff
@@ -0,0 +1,475 @@
+
+        let alive = interaction.guild.roles.cache.find((r) => r.name === "Alive")
+        let mininarr = interaction.guild.roles.cache.find((r) => r.name === "Narrator Trainee")
+        let narrator = interaction.guild.roles.cache.find((r) => r.name === "Narrator")
+        let allPlayers = [],
+            roleOptions = [],
+            allChannels = []
+        for (let i = 0; i < alive.members.size; i++) {
+            let guy = interaction.guild.members.cache.find((m) => m.nickname === (i + 1).toString())
+            if (!guy) return interaction.reply(`Player ${i} was not found!`)
+            allPlayers.push(guy.id)
+        }
+        let exists = false
+        let allchan = interaction.guild.channels.cache.filter((c) => c.name.startsWith("priv")).map((x) => x.id)
+        for (let a = 0; a < allchan.length; a++) {
+            let chan = interaction.guild.channels.cache.get(allchan[a])
+            if (chan) {
+                for (let b = 1; b <= alive.members.size; b++) {
+                    let tt = interaction.guild.members.cache.find((m) => m.nickname === b.toString())
+                    if (tt) {
+                        if (chan.permissionsFor(tt).has(["VIEW_CHANNEL"])) {
+                            exists = true
+                        }
+                    }
+                }
+            }
+        }
+        if (exists == true) {
+            interaction.reply("A player has a channel occupied already! Use `+nmanual [player number] [role]` to remove them from their channel!")
+            client.commands.get("playerinfo").run(interaction, args, client)
+            return
+        }
+
+        let revealed = interaction.guild.roles.cache.find((r) => r.name === "Revealed")
+        let bot = interaction.guild.roles.cache.find((r) => r.name === "Bots")
+        let wwsChat = interaction.guild.channels.cache.find((c) => c.name === "werewolves-chat")
+        let wwsVote = interaction.guild.channels.cache.find((c) => c.name === "ww-vote")
+        let dayChat = interaction.guild.channels.cache.find((c) => c.name === "day-chat")
+        let gamlobi = interaction.guild.channels.cache.find((c) => c.name === "game-lobby")
+        let sib = interaction.guild.channels.cache.find((c) => c.name === "sibling-chat")
+        let bandits = interaction.guild.channels.cache.find((c) => c.name === "bandits")
+        let sl = interaction.guild.channels.cache.find((c) => c.name === `sect-members`)
+        let zomb = interaction.guild.channels.cache.find((c) => c.name === "zombies")
+        let excludes = db.get("excludes") || []
+
+        let usedChannels = []
+        db.set(`usedChannels`, usedChannels)
+
+        args = roles ? roles.split(" ") : []
+        args.forEach((arg) => {
+            args[args.indexOf(arg)] = arg.toLowerCase()
+        })
+        if (args.length != alive.members.size && gamemode == "custom") {
+            return interaction.reply("The number of roles do not match the number of players!")
```

Comment:
I would suggest to make error messages ephemeral (`interaction.reply({content:"", ephemeral: true})`)

this also applies for lines 265, 269, 273, 277

Reply:
If you think this is fine as it is rn. then it would be fine

Reply:
This was intentional so the supervisors could see the messages as well

Reply:
~~Not me hopeing to hide my faults from soups~~

Reply:
Lmao